perma_web/tasks/dev.py

```diff
@@ -667,28 +668,32 @@ def populate_folder_cached_path(ctx, batch_size=500):
     else:
         logger.info("No more folders left to update!")
 
+
 #
 # Merge user accounts
 #
 
-TRANSFERRED_ORG_LINKS_CSV = 'merge_reports/tranferred_org_links.csv'
-TRANSFERRED_PERSONAL_LINKS_CSV = 'merge_reports/tranferred_personal_links.csv'
+TRANSFERRED_ORG_LINKS_CSV = 'merge_reports/transferred_org_links.csv'
+TRANSFERRED_PERSONAL_LINKS_CSV = 'merge_reports/transferred_personal_links.csv'
 MERGED_USERS_CSV = 'merge_reports/merged_users.csv'
 RETAINED_USERS_CSV = 'merge_reports/retained_users.csv'
 
-def initialize_csvs():
+
+def initialize_csvs(reports_dir):
+    p = Path(reports_dir)
+
     for filename in [TRANSFERRED_ORG_LINKS_CSV, TRANSFERRED_PERSONAL_LINKS_CSV, MERGED_USERS_CSV, RETAINED_USERS_CSV]:
-        os.makedirs(os.path.dirname(filename), exist_ok=True)
+        os.makedirs(p / os.path.dirname(filename), exist_ok=True)
```

Comment:
I haven't used the `Path` stuff yet; love this syntax.

Reply:
Hm maybe I will switch to `parent.mkdir(parents=True, exist_ok=True)` here.

Reply:
(Done in a4bab73. I might switch to pathlib for the rest of this file, but that'll be a later PR.)